docker/playwright-E2E-tests-multi-node.yml

```
@@ -0,0 +1,155 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Playwright setup for multi-node
+# ----------------------------------------------------------------------------------------------------------------------
+
+services:
+  artemis-app-node-1:
+    container_name: artemis-app-node-1
+    extends:
+      file: ./artemis.yml
+      service: artemis-app
+    image: ghcr.io/ls1intum/artemis:${ARTEMIS_DOCKER_TAG:-latest}
+    depends_on:
+      mysql:
+        condition: service_healthy
+      jhipster-registry:
+        condition: service_healthy
+      activemq-broker:
+        condition: service_healthy
+    pull_policy: always
+    restart: always
+    user: 0:0
+    env_file:
+      - ./artemis/config/prod-multinode.env
+      - ./artemis/config/node1.env
+      - ./artemis/config/cypress.env
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
```

There is a lot of duplication going on here, I think we should use [YAML anchors](https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/) to keep the code clean and short - is there a problem with YAML anchors that I am not aware of?

```suggestion
  artemis-app-base:
    &artemis-app-base
    image: ghcr.io/ls1intum/artemis:${ARTEMIS_DOCKER_TAG:-latest}
    pull_policy: always
    restart: always
    user: 0:0  # Consider using a non-root user
    depends_on:
      mysql:
        condition: service_healthy
      jhipster-registry:
        condition: service_healthy
      activemq-broker:
        condition: service_healthy
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock

  artemis-app-node-1:
    <<: *artemis-app-base
    container_name: artemis-app-node-1
    env_file:
      - ./artemis/config/prod-multinode.env
      - ./artemis/config/node1.env
      - ./artemis/config/cypress.env
```

---

didnt about the anchors! done 👍

---

Ah I didn't know about the anchors! But it seems like it doesn't work with compose 😕